autovacuum_message_attachment/models/autovacuum_mixin.py

```diff
@@ -54,7 +54,7 @@ def _get_autovacuum_records_model(self, rule):
         )
         autovacuum_relation = self._autovacuum_relation
         for leaf in domain:
-            if not isinstance(leaf, (tuple, list)):
+            if not isinstance(leaf, (tuple | list)):
```

Comment:
why switching from `(tuple, list)` to `(tuple | list)` ?
note: the latter works only in Python 3.10+, the recommended version for Odoo 18.0)

Reply:
Linter fix:
```
autovacuum_message_attachment/models/autovacuum_mixin.py:57:20: UP038 Use `X | Y` in `isinstance` call instead of `(X, Y)`
```